crypto/cms/cms_smime.c

```diff
@@ -314,9 +314,13 @@ int CMS_verify(CMS_ContentInfo *cms, STACK_OF(X509) *certs,
     /* Attempt to verify all signers certs */
 
     if (!(flags & CMS_NO_SIGNER_CERT_VERIFY) || (flags & CMS_CADES)) {
-        if (flags & CMS_CADES)
+        if (flags & CMS_CADES) {
```

Comment:
@matt told me in my last PR that these tests with & should be flags&X != 0.
There are more of these in this commit